packages/core/src/scss/components/checkout/paymentProvider/_paymentProvider.scss

```diff
@@ -5,17 +5,24 @@
     align-items: start;
     display: flex;
     justify-content: space-between;
+    flex-wrap: wrap;
+}
+
+.paymentProviderHeader-subtitleContainer {
+    flex-basis: 100%;
 }
 
 .paymentProviderHeader-nameContainer {
     display: flex;
     justify-content: space-between;
+    flex-basis: 50%;
 }
 
 .paymentProviderHeader-img {
     align-self: center;
     display: flex;
     height: fontSize("large");
+    object-fit: contain;
```

Comment:
Does the flex layout change the look of the header img?

Reply:
Yes